suite-native/atoms/src/Table.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type TableCell = string | ReactNode;
+
+type TableRowProps = {
+    rowData: TableCell[];
+    rowIndex: number;
+    highlightFirstRow: boolean;
+    highlightFirstCol: boolean;
+};
+type TableProps = {
+    dataCells: TableCell[][];
+    highlightFirstRow?: boolean;
+    highlightFirstCol?: boolean;
+};
+
+const TableRow = ({ rowData, rowIndex, highlightFirstRow, highlightFirstCol }: TableRowProps) => (
+    <Box flexDirection="row" justifyContent="space-between" marginVertical="small">
+        {rowData.map((cell, colIndex) => {
+            const isHeaderCell = rowIndex === 0;
+            const isTitleCell = colIndex === 0;
+            const cellIsHighlighted =
+                (highlightFirstRow && isHeaderCell) || (highlightFirstCol && isTitleCell);
+            return (
+                // eslint-disable-next-line react/no-array-index-key
+                <Box flex={1} key={`${rowIndex}:${colIndex}`}>
+                    <Text variant="hint" color={cellIsHighlighted ? 'gray600' : 'gray1000'}>
+                        {cell}
+                    </Text>
+                </Box>
+            );
+        })}
+    </Box>
+);
+
+export const Table = ({
+    dataCells,
+    highlightFirstRow = true,
+    highlightFirstCol = true,
+}: TableProps) => (
+    <Box>
+        {dataCells.map((row, rowIndex) => (
+            <TableRow
+                // eslint-disable-next-line react/no-array-index-key
+                key={rowIndex}
```

Comment:
This rule is there for a reason. I would not disable it and found better way to get key. This is pretty big antipattern and could be harmful potentially 

https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

Reply:
Could you please point me out what else could I use for `key`? Since this component is taking generic data, I don't see any other way how to calculate truly unique values.

Reply:
@juriczech is right about it, you shouldn't use index unless you are very aware what you are doing. Using index as key could lead to some random behavior like data !== UI and so on. If you don't have anything that you can use, even `Math.random` is better than index.

But this is not that case and all issue could be solved if you refactor this component to work same as tables in plain HTML so in place of usage you will import everything yourself and you will just do:

```tsx
<Table>
  <TableRow>
    <Td>{something}</Td>
  </TableRow>
</Table>
```

If you do this you won't need any `key` at all, and component is also much more customizable.

One more advice, when designing new "atom" component, I strongly recommend you to check how component API is done in other UI lib, which is usually very well designed. My favorites are CharkaUI (they have Table component https://chakra-ui.com/docs/components/table/usage) and another very nice is Tamagui (https://tamagui.dev/docs/intro/introduction).